__tests__/helpers/Signatures.bs.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+
+var Mappable = {};
+
+var Applicative = {};
+
+var Monad = {};
+
+var Reduceable = {};
+
+var Iterable = {};
+
+var Concatenable = {};
+
+exports.Mappable = Mappable;
+exports.Applicative = Applicative;
+exports.Monad = Monad;
+exports.Reduceable = Reduceable;
+exports.Iterable = Iterable;
+exports.Concatenable = Concatenable;
+/* No side effect */
```

Comment:
This is why I preferred using stand-alone interface files when only interface definitions are needed, until support for that was dropped. Implementation files generate unnecessary code, interface files don't.

Reply:
Interesting. I wonder if there is a flag for that.

Reply:
No, I think it's disallowed because the build system needs a source file for invalidation or some other such technical reason. It's very deliberate at least, and it's a restriction imposed by BuckleScript, not OCaml.